src/creature.h

```diff
@@ -158,7 +158,7 @@ class Creature : virtual public Thing
 		virtual bool canSeeCreature(const Creature* creature) const;
 
 		virtual RaceType_t getRace() const {
-			return RACE_NONE;
+			return RACE_BLOOD;
```

Comment:
Noob question, creature will always be of type blood?

Reply:
No, its overrided by xml for monsters. Players and npc yes, unless you have overrided function or specific cases (like monsters in the xml). But now the default is blood instead of none (since none is kinda useless, has no effects defined). Its preferrable to creatures defautly bleed than have weird black hit point.

Reply:
nice, tks!